fe/fe-core/src/main/java/com/starrocks/connector/paimon/PaimonMetadata.java

```diff
@@ -179,6 +180,10 @@ public Table getTable(String dbName, String tblName) {
             LOG.error("Get paimon table {}.{} createtime failed, error: {}", dbName, tblName, e);
         }
         PaimonTable table = new PaimonTable(this.catalogName, dbName, tblName, fullSchema, paimonNativeTable, createTime);
+        if (!(paimonNativeTable instanceof AbstractFileStoreTable ||
+                paimonNativeTable instanceof ReadOptimizedTable)) {
+            table.setSystemTable(true);
```

Comment:
refer to the `AbstractCatalog::isSpecifiedSystemTable` in the Paimon project to set this variable